### PR TITLE
Add RLGym to project page

### PR DESCRIPTION
**Pull request by AechPro**
Per the request in [this issue](https://github.com/lucas-emery/rocket-league-gym/issues/24) on our repo, we have submitted an addition to the supported projects page.

### CODE REVIEW

**Comment by araffin** [comment]
Hello,
thanks for the PR =)
please update the changelog and we are good to go ;)

**Comment by AechPro** [comment]
That should do it!

File: docs/misc/projects.rst [hunk 1]
```diff
@@ -74,3 +74,17 @@ SuperSuit contains easy to use wrappers for Gym (and multi-agent PettingZoo) env
...
+
+Rocket League Gym
+-----------------
+A fully custom python API and C++ DLL to treat the popular game Rocket League like an OpenAI Gym environment.
+
+- Dramatically increases the rate at which the game runs.
+- Supports full configuration of initial states, observations, rewards, and terminal states.
+- Supports multiple simultaneous game clients.
+- Supports multi-agent training and self-play.
+- Provides custom wrappers for easy use with stable-baslines3.
```
Comment (araffin) [inline docs/misc/projects.rst:86]:
```suggestion
- Provides custom wrappers for easy use with stable-baselines3.
```

**Review by araffin — APPROVED** [review]
LGTM, thanks =)

**Pull request merged**